### PR TITLE
Feat/grouped contacts icon

### PR DESCRIPTION
**Pull request by y-golde**
Moved grouped investigation state to redux (in order to use the state in both contact questioning and in interactions tab)

added new icon for grouped contact.

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    const fetchGroupedInvestigationContacts = (groupId : string) => {
+        return axios.get<ConnectedInvestigationContact>(`/intersections/groupedInvestigationsContacts/${groupId}`)
+            .then((response) => {
+                return response.data;
+            })
+            .catch((error) => {
+                return {
+                    investigationGroupReasonByReason: {
+                        displayName : "טוען..."
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts:34]: replace with ingle quotes

File: client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    const setGroupedInvestigationsDetailsAsync = async () => {
+        const groupId = await fetchGroupedId();
+        setGroupId(groupId);
+        if(groupId) {
+            const investigations = await fetchGroupedInvestigationContacts(groupId);
+            setGroupedInvestigations(investigations);
+        }
+
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts:50]: delete empty row

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsActionTypes.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import ConnectedInvestigationContact from 'models/GroupedInvestigationContacts/ConnectedInvestigationContact';
+
+export const SET_GROUP_ID = 'SET_GROUP_ID';
+export const SET_GROUPED_INVESTIGATIONS = 'SET_GROUPED_INVESTIGATIONS';
```
Comment (Noam6421) [inline client/src/redux/GroupedInvestigations/GroupedInvestigationsActionTypes.ts:4]: add empty row

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import ConnectedInvestigationContact from 'models/GroupedInvestigationContacts/ConnectedInvestigationContact';
+
+interface GroupedInvestigationReducerType {
+    groupId : string,
+    investigations : ConnectedInvestigationContact
+}
+
+export const initialState : GroupedInvestigationReducerType ={
+    groupId : "",
```
Comment (Noam6421) [inline client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts:9]: single quotes

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    investigations : {
+        investigationGroupReasonByReason: {
+            displayName : "טוען..."
```
Comment (Noam6421) [inline client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts:12]: same

**Review by Noam6421 — CHANGES_REQUESTED** [review]

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts (hunk 5 above)
Comment (y-golde) [reply client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1255/commits/e0ec564a60582787ff95dc9832adeafe29c253e3)

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts (hunk 4 above)
Comment (y-golde) [reply client/src/redux/GroupedInvestigations/GroupedInvestigationsType.ts]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1255/commits/e0ec564a60582787ff95dc9832adeafe29c253e3)

File: client/src/redux/GroupedInvestigations/GroupedInvestigationsActionTypes.ts (hunk 3 above)
Comment (y-golde) [reply client/src/redux/GroupedInvestigations/GroupedInvestigationsActionTypes.ts:4]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1255/commits/e0ec564a60582787ff95dc9832adeafe29c253e3)

File: client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts (hunk 2 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1255/commits/e0ec564a60582787ff95dc9832adeafe29c253e3)

File: client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/useGroupedInvestigationContacts.ts]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1255/commits/e0ec564a60582787ff95dc9832adeafe29c253e3)

**Review by Noam6421 — DISMISSED** [review]

**Review by chenFriedman — APPROVED** [review]

**Review by Noam6421 — APPROVED** [review]

**Pull request merged**